Umbraco-Heartcore/index.md

```diff
@@ -0,0 +1,37 @@
+---
+versionFrom: 8.0.0
+---
+
+# Umbraco Heartcore
+
+In this section you will find documentation on how to work with Umbraco Heartcore.
+
+It includes API documentation, the basics on how to get started and how to work with the available Client Libraries.
+
+Please note that Umbraco Heartcore is a specific type of project only available via Umbraco Cloud.
+
+## What is Umbraco Heartcore?
+
+Umbraco Heartcore is a SaaS (Software as a Service) offered by Umbraco. The service enables you to create and manage content and media in the Umbraco backoffice and make it available to any - and multiple - platforms, devices, channels etc. in order to distribute your content. 
+
+The product comes with a set of API endpoints that enables you to provide content through Umbraco to websites, apps, smartwatches or any other platform you can think of. The idea is that you create, manage and work with content in the Umbraco backoffice and then you use the API endpoints to distribute the content wherever you want.
+
+You can sign up to be first in line for more details on the product on [Umbraco.com/Headless](https://umbraco.com/headless/).
+
+## [API Documentation for the Umbraco Heartcore REST API endpoints](API-Documentation/)
+
+Reference documentation for the two APIs available, as well as details about common HTTP headers, versioning, REST Standard and how to work with authentication and authorization.
+
+- [Authentication and Authorization](API-Documentation/#authentication-and-authorization)
+- [Content Delivery API](API-Documentation/Content-Delivery)
+- [Content Management API](API-Documentation/Content-Management)
+
+## [Getting Started with Umbraco Heartcore](Getting-Started-Cloud/)
+
+In order to get started with Umbraco Heartcore you will need to create a new project via the Umbraco Cloud Portal and familiarize yourself with the features available. This includes the Heartcore section in the backoffice and creating API Keys for authentication and authorization purposes.
```

Comment:
```suggestion
In order to get started with Umbraco Heartcore you will need to create a new project via the Umbraco Cloud Portal and familiarize yourself with the features available. This includes the Umbraco Heartcore section in the backoffice and creating API Keys for authentication and authorization purposes.
```

Reply:
Ah, this one is actually the section in the backoffice currently named 'Headless' - so, should we hold off renaming that for now, like you've suggested other places?

Reply:
Ahhh, yes. Let's leave it as Headless for now, I've updated the suggestion